buildres/mac/jabrefHost.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
```

Comment:
It is not possible to reuse the linux version?

Reply:
It is basically the same file. The issue is that with the --resource-dir option in jpackage I cannot add a single file from another directory.
So I copied it and changed the path settings.